jbpm-wb-integration/jbpm-wb-integration-backend/src/test/java/org/jbpm/workbench/wi/backend/server/dd/DDEditorServiceTest.java

```diff
@@ -262,6 +262,36 @@ public void testValidateMvelResolver() {
     }
 
     @Test
+    public void testValidateMvelResolverWithSupportedParameters() {
+        List<NamedObjectModel> models = Arrays.asList(new NamedObjectModel[]{
+                new NamedObjectModel(ItemObjectModel.MVEL_RESOLVER,
+                                     "item-name",
+                                     "java.util.Arrays.asList(ksession, taskService, runtimeManager, classLoader, entityManagerFactory);")});
+
+        List<ValidationMessage> validationMessages = validateObjectModels(null, models);
+        assertEquals(0, validationMessages.size());
+
+    }
+
+    @Test
+    public void testValidateMvelResolverWithUnSupportedParameters() {
+        List<NamedObjectModel> models = Arrays.asList(new NamedObjectModel[]{
+                new NamedObjectModel(ItemObjectModel.MVEL_RESOLVER,
+                                     "item-name",
+                                     "java.util.Arrays.asList(unsupportedParamName);")});
+
+        List<ValidationMessage> validationMessages = validateObjectModels(null, models);
+        assertEquals(1, validationMessages.size());
```

Comment:
Similar situation as above, only this time with `Assertions.assertThat(validationMessages).hasSize(1)`.